libs/features/mas/src/merch-card.js

```diff
@@ -8,6 +8,8 @@ import {
 
 import './global.css.js';
 import './aem-fragment.js';
+import './merch-mnemonic-list.js';
+import './merch-whats-included.js';
```

Comment:
I wonder what is the impact of including `merch-whats-included.js in` bytes.
if we render only on clicks, we could also load it on the fly.

Reply:
4 bytes :)?

Reply:
not sure, we need to keep in mind that many cards on adobe.com do not need this, especially on page load.